### ISSUE TITLE
igc ingress command works in IKS but not OCP

### ISSUE THREAD

**Issue by bwoolf1** [ibm-garage-cloud/planning#122]
The `igc ingress` command works in IKS but not in OCP 3.11 nor OCP 4.3.

When logged into an IKS cluster, the command works:

```
$ igc ingress
? Ingresses in the 'dev' namespace. Select an ingress to launch the default browser or 'Exit'.
 exit
  template-node-typescript - https://template-node-typescript-dev.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
```
```
$ igc ingress -n tools
? Ingresses in the 'tools' namespace. Select an ingress to launch the default browser or 'Exit'.
 exit
  argocd-server - https://grpc.argocd.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
  argocd-server - https://argocd.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
  artifactory - https://artifactory.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
  catalyst-dashboard - https://dashboard.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
  jenkins - https://jenkins.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
  pact-broker - https://pact.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
  sonarqube - https://sonarqube.garage-dev-iks-cl-201185-0143c5dd31acd8e030a1d6e0ab1380e3-0000.us-east.containers.appdomain.cloud
```

However, when logged into an OCP cluster (3.11 or 4.3), the command stops working:

For example:
```
$ igc ingress -n tools
No ingresses found for the 'tools' namespace
```

This is even though the Dashboard has tiles for tools like Artifactory and SonarQube, and the tiles open the tools' dashboards.

Likewise, the dev namespace has a route called template-node-typescript to a service with the same name (but no ingress), yet:
```
$ igc ingress         
No ingresses found for the 'dev' namespace
```

igc ingress needs to work with both ingress pods and routes.

**Comment by seansund** [comment]
We have this recorded multiple times -  #35 , ibm-garage-cloud/ibm-garage-cloud-cli#11 , and #92

**Comment by seansund** [comment]
Addressed in 0.4.8 of the cli

**Comment by seansund** [comment]
This should be fixed now @bwoolf1  Can you re-test with cli > v0.4.8?
